line-sdk/src/main/java/com/linecorp/linesdk/LineFriendProfile.java

```diff
@@ -0,0 +1,119 @@
+package com.linecorp.linesdk;
+
+import android.net.Uri;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+/**
+ * Represents a friend's LINE profile in the Social API.
+ */
+public class LineFriendProfile extends LineProfile {
+    public static final Parcelable.Creator<LineFriendProfile> CREATOR =
+            new Parcelable.Creator<LineFriendProfile>() {
+                @Override
+                public LineFriendProfile createFromParcel(final Parcel in) {
+                    return new LineFriendProfile(in);
+                }
+
+                @Override
+                public LineFriendProfile[] newArray(final int size) {
+                    return new LineFriendProfile[size];
+                }
+            };
+
+    /**
+     * Friend's nickname which changed by owner user
+     */
+    @Nullable
+    private final String displayNameOverridden;
```

Comment:
The variable should end with Noun.
Please help to modify `displayNameOverridden` to `OverriddenDisplayName`, and also for other cases too.

Reply:
the property name in LINE API response is `"displayNameOverridden"`
do you prefer use a **different** property name in SDK?

I am not sure, but which approach is better?
Should we use **same property name** in SDK, or should we use **correct** one?

Reply:
I think it's fine to just change the name in LINE SDK only, so that when app developer use LINE SDK, he can get a variable name that is more fitting the naming convention.

Most developers won't dig into what Web API we actually called inside LINE SDK; so it should be fine if we don't provide exact same names according to the response data.

Reply:
fixed